WISE-Community/WISE-Client
# chore(Authoring): Convert project asset authoring to Angular

## Changes

Converted project asset authoring to Angular.
Note: We will delete ProjectAssetService.openAssetChooser() after we convert ProjectInfoAuthoringController to Angular because that view still uses the old asset chooser.

## Test

- Make sure the asset authoring view still works properly
- Make sure the select asset popup still works properly (like when choosing an image in various places like draw background image, match choice, multiple choice choice, html component insert asset, etc.)
- Make sure drag and drop as well as click to open file chooser both work
- Make sure you can only upload accepted file types

Closes #1337
